chore: remove and rename unused  pearson vue packages

Ang-m4 opened the pull request:
## Description

This PR removes all unused packages from the `eox_nelp` project that were migrated to the new Pearson Vue Engine API.  

- First, I identified and removed deprecated files, along with their import and test references, in commit [fe1f485](https://github.com/eduNEXT/eox-nelp/pull/249/commits/fe1f485bec00a37e1f252bce5702771311b670ec).  
- Then, after updating the old `pearson_vue` package, I renamed it to `pearson_vue_engine` in commit [92e79a6](https://github.com/eduNEXT/eox-nelp/pull/249/commits/92e79a670579fa64150c0c3fcf3685915c2e2adc).  

## Migrations

- The `PearsonRTENEvent` model was deleted, and its corresponding [migration](https://github.com/eduNEXT/eox-nelp/pull/249/files#diff-44fc633b9d411c7e9eedfde34126380181c5a0b77ee188147d1fc9c5c4d97b20) was generated.

## How to test:

### Prerequisites  
Make sure you have the following settings updated in the `TenantConfig` you are testing with:  
![Settings Configuration - Pearson Vue](https://github.com/user-attachments/assets/78679080-5ffd-44f8-a41a-84241acd9c7c)  

1. The course you are testing must be added to the `PEARSON_ENGINE_COURSES_ENABLED` attribute.  
2. The `PEARSON_RTI_TESTING_SKIP_HANDLE_COURSE_COMPLETION_STATUS` setting must be set to `true`.  

### Steps to Test  

1. Enroll a user in the course you are testing.  
2. Navigate to the **CourseEnrollment** admin site:  
   ![Course Enrollment Admin](https://github.com/user-attachments/assets/9ce7ee72-4c25-46ef-84bc-98a35448dcfd)  
3. Select the corresponding **CourseEnrollment** record.  
4. Open the **Actions** dropdown and select the type of request you need.  
5. Click **Go** to perform the request.  

### Verifying the Request  

- To check if the request was successful, go to the **AuditModel** admin section and verify the request status.  
  ![AuditModel Request Verification](https://github.com/user-attachments/assets/9703842b-34fd-4c05-8bf5-174fa5838076)  

---
## Rebase update

After the rebase update i made to this PR, the requests functionality seems to be working just fine.

![pearson-vue-rebase-update](https://github.com/user-attachments/assets/7a4ac665-9826-4981-bdf4-94d57f8422b9)

andrey-canon reviewed (commented):
I tested main functionality  and it's working as expect just one comment

Ang-m4 commented:
Hey @andrey-canon,

Just a quick update on the PR:

- I rebased last week's changes to the `pearson_vue` package. Most things are still the same.
- Noticed the `__init__.py` file was missing in the `tests` folder (which was skipping all tests), so I added it.
- Removed the unused pylint disable from the constants files, as you suggested.

andrey-canon reviewed (approved):
LGTM, please rebase, merge and test it in stage
